core/schemas/package.py

```diff
@@ -0,0 +1,276 @@
+import json
+from datetime import datetime, timezone
+from typing import Any, ClassVar, Dict, List, Optional, Tuple
+
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Self
+
+from core.schemas import entity, indicator, observable
+from core.schemas.observable import ObservableTypes
+
+
+class YetiPackageRelationship(BaseModel):
+    target: str
+    link_type: str = "observes"
+
+
+class YetiPackage(BaseModel):
+    timestamp: str | int  # add validator
```

Comment:
Maybe we should settle on one type here? Why not a python datetime actually?

Reply:
How is it handled when building from json with datetime strings?

Reply:
fixed in ccfd48efcef7333a5118800b4dcf12ccd0372594 with timestamps tests